static/app/views/issueDetails/groupDetails.tsx

```diff
@@ -188,25 +180,12 @@ class GroupDetails extends Component<Props, State> {
     return `/issues/${this.props.params.groupId}/first-last-release/`;
   }
 
-  async getEvent(group?: Group) {
-    if (group) {
-      this.setState({loadingEvent: true, eventError: false});
```

Comment:
my thinking was that the `loadingEvent` state is set to true here which affects `refetchGroup` but because we don't rely on the group to make this call anymore, removing the setState shouldn't have an effect on if the group if refetched.

Reply:
Looks like fetching the. next event still work, but `loadingEvent` does get passed down to `GroupEventDetails` where it gets used to show a loading spinner. So without this logic, the UI won't change to show the loading state when scrolling through events